src/modules/session-storage/SessionFileStorage.ts

```diff
@@ -25,20 +26,24 @@ export class SessionFileStorage implements ISessionStorage {
   // biome-ignore lint/suspicious/noExplicitAny: <explanation>
   private async readDataFromFile(type: "sessions" | "signers"): Promise<any> {
     return new Promise((resolve) => {
-      // @ts-ignore
-      fs.readFile(this.getStorageFilePath(type), "utf8", (err, data) => {
-        if (err) {
-          // Handle errors appropriately
-          resolve(undefined)
-        } else {
-          if (!data) {
-            resolve(null)
+      nodeFs.readFile(
```

Comment:
This will only work on backend, which is expected as this is for file storage. Is there any possibility of FE getting error because one of the files is using node-fs?

Reply:
Not unless the dev explicitly does something silly. By default localStorage only is used in the browser.